src/components/WalletConnect.jsx

```diff
@@ -67,10 +67,20 @@ export default function WalletConnect() {
                       dispatch(setWallet({ address: displayWallet() }));
                       const owner_address = getWallet().address;
                       let in_DB = await userInDB(owner_address);
+                      let in_Totals = await userInTotals(owner_address);
```

Comment:
Would this look something like:

let inTotalsPromise = new Promise(userInTotals(owner_address));
and then 
let in_totals = await inTotalsPromise;
on line 70?
If so, would that really make it any faster? I think it the promise would still be invoked and fulfilled every time because I can't think of a scenario in which it wouldn't check if the user is in the database.

Reply:
ok sounds good, will do!